components/formats-gpl/src/loci/formats/in/MetamorphReader.java

```diff
@@ -1361,6 +1361,8 @@ private String getRealSTKFile(Location l) {
     String parent = l.getParent();
 
     if (name.indexOf('_') > 0) {
+      String pattern = "_[swt]\\d+";
+      System.out.println(name.replaceAll(pattern, ""));
```

Comment:
Can this `System.out.println` be removed (or replaced using logging at a debug level minimally) as it clutters the console of the automated builds?